src/sshping.cxx

```diff
@@ -127,13 +142,100 @@ void die(const char* msg) {
     exit(255);
 }
 
+#if _WIN32
```

Comment:
Formatting and consistency:

```
#ifdef _WIN32
#include <windows.h>
  #include <conio.h>
  #ifndef PASS_MAX
    #define PASS_MAX 512
  #endif
```

`#if` and `#ifdef` logic is quite different. Although `#if defined()` and `#ifdef` are quite similar. Best be consistent. #ifdef is more commonly used in this codebase and in general.